File: Album/src/main/java/gorden/album/item/ItemCamera.java
package gorden.album.item;

import android.content.Context;
import android.graphics.Color;
import android.support.v4.content.ContextCompat;
import android.view.Gravity;
import android.widget.ImageView;
import android.widget.LinearLayout;
import android.widget.TextView;

import gorden.album.R;

/**
 * 相机
 * Created by Gorden on 2017/4/2.
 */

public class ItemCamera extends LinearLayout{
    public ItemCamera(Context context) {
        super(context);
        initView(context);
    }

    private void initView(Context context) {
        setGravity(Gravity.CENTER);
        setOrientation(VERTICAL);
        setBackgroundColor(Color.parseColor("#66000000"));
        ImageView imageView = new ImageView(context);
        imageView.setImageResource(android.R.drawable.ic_menu_camera);
        addView(imageView,dip2px(40,context),dip2px(40,context));

        TextView textView = new TextView(context);
        textView.setText("拍摄照片");
        textView.setTextColor(ContextCompat.getColor(context, R.color.album_textcolor));
        textView.setTextSize(14);
        addView(textView,LayoutParams.WRAP_CONTENT,LayoutParams.WRAP_CONTENT);
    }

    public static int dip2px(int dip, Context context) {
        return (int) (dip * context.getApplicationContext().getResources().getDisplayMetrics().density + 0.5f);
    }
}

File: lib_album/src/main/kotlin/gorden/library/item/ItemCamera.kt
package gorden.library.item

import android.content.Context
import android.graphics.Color
import android.support.v4.content.ContextCompat
import android.view.Gravity
import android.widget.ImageView
import android.widget.LinearLayout
import android.widget.TextView

import gorden.library.R
import gorden.library.entity.dp


/**
 * 相机
 */

class ItemCamera(context: Context) : LinearLayout(context) {
    init {
        gravity = Gravity.CENTER
        orientation = LinearLayout.VERTICAL
        setBackgroundColor(Color.parseColor("#66000000"))
        val imageView = ImageView(context)
        imageView.setImageResource(android.R.drawable.ic_menu_camera)
        addView(imageView, 40.dp,40.dp)

        val textView = TextView(context)
        textView.text = "拍摄照片"
        textView.setTextColor(ContextCompat.getColor(context, R.color.album_textcolor))
        textView.textSize = 14f
        addView(textView, LinearLayout.LayoutParams.WRAP_CONTENT, LinearLayout.LayoutParams.WRAP_CONTENT)
    }
}
